### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by fayazara**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items

### CODE REVIEW

**Comment by davemachado** [comment]
couple things:

- remove extra whitespace at end of lines
- remove extra newlines at end of sections (should be 1, not 2)
- your fork is not up-to-date with the current `mater`. Please rebase so that your diff does not include regressions on entries you are not updating (labstack, for example).

**Comment by fayazara** [comment]
@davemachado Have updated, as commented above.

File: README.md [hunk 1]
```diff
@@ -100,7 +100,6 @@ API | Description | Auth | HTTPS | CORS | Link |
 ### Books
 API | Description | Auth | HTTPS | CORS | Link |
 |---|---|---|---|---|---|
-| Bhagavad Gita | Bhagavad Gita text | `OAuth` | Yes | Yes | [Go!](https://bhagavadgita.io/api) |
```
Comment (davemachado) [inline README.md]: Why remove this?

File: README.md [hunk 2]
```diff
@@ -151,23 +150,27 @@ API | Description | Auth | HTTPS | CORS | Link |
 API | Description | Auth | HTTPS | CORS | Link |
 |---|---|---|---|---|---|
 | Binance | Exchange for Trading Cryptocurrencies based in China | `apiKey` | Yes | Unknown | [Go!](https://github.com/binance-exchange/binance-official-api-docs) |
+| BitBNS | Indian Cryptocurrency exchange | No | Yes | No | [Go!](https://bitbns.com/order/getTickerAll) |
```
Comment (davemachado) [inline README.md]: Please link to documentation rather than an endpoint.

File: README.md [hunk 3]
```diff
@@ -151,23 +150,27 @@ API | Description | Auth | HTTPS | CORS | Link |
...
 | BitcoinCharts | Financial and Technical Data related to the Bitcoin Network | No | Yes | Unknown | [Go!](https://bitcoincharts.com/about/exchanges/) |
 | Bitfinex | Cryptocurrency Trading Platform | `apiKey` | Yes | Unknown | [Go!](https://docs.bitfinex.com/docs/introduction) |
 | Bitmex | Real-Time Cryptocurrency derivatives trading platform based in Hong Kong | `apiKey` | Yes | Unknown | [Go!](https://www.bitmex.com/app/apiOverview) |
 | Bittrex | Next Generation Crypto Trading Platform | `apiKey` | Yes | Unknown | [Go!](https://bittrex.com/Home/Api) |
 | Block | Bitcoin Payment, Wallet & Transaction Data | `apiKey` | Yes | Unknown | [Go!](https://www.block.io/docs/basic) |
 | Blockchain | Bitcoin Payment, Wallet & Transaction Data | No | Yes | Unknown | [Go!](https://www.blockchain.info/api) |
+| Blockcypher | Cryptocurrency transaction status API | No | Yes | Unknown | [Go!](https://www.blockcypher.com/dev/) |
```
Comment (davemachado) [inline README.md]: remove `API` from Desc.

File: README.md [hunk 4]
```diff
@@ -151,23 +150,27 @@ API | Description | Auth | HTTPS | CORS | Link |
...
 | CoinAPI | All Currency Exchanges integrate under a single api | `apiKey` | Yes | No | [Go!](https://docs.coinapi.io/) |
 | Coinbase | Bitcoin, Litecoin and Ethereum Prices | `apiKey` | Yes | Unknown | [Go!](https://developers.coinbase.com) |
 | CoinBin | Cryptocurrency information | No | Yes | Unknown | [Go!](https://coinbin.org/) |
 | CoinDesk | Bitcoin Price Index | No | No | Unknown | [Go!](http://www.coindesk.com/api/) |
 | CoinMarketCap | Cryptocurrencies Prices | No | Yes | Unknown | [Go!](https://coinmarketcap.com/api/) |
 | CryptoCompare | Cryptocurrencies Comparison | No | Yes | Unknown | [Go!](https://www.cryptocompare.com/api#) |
-| Cryptonator | Cryptocurrencies Exchange Rates | No | Yes | Unknown | [Go!](https://www.cryptonator.com/api/) |
+| Cryptonator | Cryptocurrencies Exchange Rates | No | Yes | Unknown | [Go!](https://www.cryptonator.com/api/) |
```
Comment (davemachado) [inline README.md]: strip whitespace - this is not a change.

File: README.md [hunk 5]
```diff
@@ -151,23 +150,27 @@ API | Description | Auth | HTTPS | CORS | Link |
...
 | CoinAPI | All Currency Exchanges integrate under a single api | `apiKey` | Yes | No | [Go!](https://docs.coinapi.io/) |
 | Coinbase | Bitcoin, Litecoin and Ethereum Prices | `apiKey` | Yes | Unknown | [Go!](https://developers.coinbase.com) |
 | CoinBin | Cryptocurrency information | No | Yes | Unknown | [Go!](https://coinbin.org/) |
 | CoinDesk | Bitcoin Price Index | No | No | Unknown | [Go!](http://www.coindesk.com/api/) |
 | CoinMarketCap | Cryptocurrencies Prices | No | Yes | Unknown | [Go!](https://coinmarketcap.com/api/) |
 | CryptoCompare | Cryptocurrencies Comparison | No | Yes | Unknown | [Go!](https://www.cryptocompare.com/api#) |
-| Cryptonator | Cryptocurrencies Exchange Rates | No | Yes | Unknown | [Go!](https://www.cryptonator.com/api/) |
+| Cryptonator | Cryptocurrencies Exchange Rates | No | Yes | Unknown | [Go!](https://www.cryptonator.com/api/) |	
 | GDAX | Cryptocurrency Trading Platform | `apiKey` | Yes | Unknown | [Go!](https://docs.gdax.com/#api) |
+| Koinex | Indian Cryptocurrency exchange | No | Yes | Yes | [Go!](https://koinex.in/api/ticker) |
```
Comment (davemachado) [inline README.md]: link to documentation not an endpoint.

File: README.md [hunk 6]
```diff
@@ -176,12 +179,12 @@ API | Description | Auth | HTTPS | CORS | Link |
 | Currencylayer | Exchange rates and currency conversion | `apiKey` | Yes | Unknown | [Go!](https://currencylayer.com/documentation) |
 | Czech National Bank | A collection of exchange rates | No | Yes | Unknown | [Go!](https://www.cnb.cz/cs/financni_trhy/devizovy_trh/kurzy_devizoveho_trhu/denni_kurz.xml) |
 | Fixer.io | Exchange rates and currency conversion | No | Yes | Unknown | [Go!](http://fixer.io) |
-| LabStack | Accurate and reliable live currency exchange rates for over 150 symbols | `apiKey` | Yes | Yes | [Go!](https://labstack.com/docs/api/currency) |
```
Comment (davemachado) [inline README.md]: All these LabStack changes show your fork is stale, unless you purposefully changed these links. Please update based on this project as origin branch.

File: README.md [hunk 7]
```diff
@@ -467,6 +471,8 @@ API | Description | Auth | HTTPS | CORS | Link |
 | NPR One | Personalized news listening experience from NPR | `OAuth` | Yes | Unknown | [Go!](http://dev.npr.org/api/) |
 | The Guardian | Access all the content the Guardian creates, categorised by tags and section | `apiKey` | Yes | Unknown | [Go!](http://open-platform.theguardian.com/) |
 
+
```
Comment (davemachado) [inline README.md]: remove extra WS.

File: README.md [hunk 8]
```diff
@@ -756,7 +763,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | ODWeather | Weather and weather webcams | No | No | Unknown | [Go!](http://api.oceandrivers.com/static/docs.html) |
 | OpenUV | Real-time UV Index Forecast | `apiKey` | Yes | Unknown | [Go!](https://www.openuv.io) |
 | OpenWeatherMap | Weather | `apiKey` | No | Unknown | [Go!](http://openweathermap.org/api) |
-| Storm Glass | Global marine weather from multiple sources | `apiKey` | Yes | Yes | [Go!](https://stormglass.io/) |
 | Weatherbit | Weather | `apiKey` | Yes | Unknown | [Go!](https://www.weatherbit.io/api) |
 | Wunderground | Weather | `apiKey` | Yes | Unknown | [Go!](https://www.wunderground.com/weather/api/) |
 | Yahoo! Weather | Weather | No | Yes | Unknown | [Go!](https://developer.yahoo.com/weather/) |
+
```
Comment (davemachado) [inline README.md]: remove

File: README.md [hunk 9]
```diff
@@ -522,7 +528,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 ### Personality
 API | Description | Auth | HTTPS | CORS | Link |
 |---|---|---|---|---|---|
-| chucknorris.io | JSON API for hand curated Chuck Norris jokes | No | Yes | Unknown | [Go!](https://api.chucknorris.io) |
+| chucknorris.io | JSON API for hand curated Chuck Norris jokes | No | Yes | Unknown | [Go!](https://api.chucknorris.io) |
```
Comment (davemachado) [inline README.md]: remove

**Comment by davemachado** [comment]
Bump

**Pull request closed without merging**